encoding.go

```diff
@@ -29,7 +29,8 @@ const (
 	Protobuf    = "protobuf"
 	Text        = "text"
 	TextNewline = "textnl"
-	CLI         = "cli"
+
+	CLI = "*cli.responseEmitter"
```

Comment:
what?

Reply:
Oh yes, I thought about finding a better way to do this.

Now that cli.Run doesn't know very much about the executor, the only way to check which postrun to use is to look at the ResponseEmitter. This was the simplest way to do it.
I can also add a `type Namer interface { Name() string }`, make `*cli.responseEmitter` implement that to return `"cli"` and change this line back.

Why didn't I have that idea earlier again?